assets/src/components/clipboard-button/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef, useEffect } from '@wordpress/element';
+import { useCopyToClipboard } from '@wordpress/compose';
+import { Button } from '@wordpress/components';
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+const TIMEOUT = 4000;
+
+// Migrated from @wordpress/components.
+export default function ClipboardButton( {
```

Comment:
I suppose `wp.compose.useCopyToClipboard` not being used to have more compatibility with older versions of Gutenberg?

Reply:
Our `ClipboardButton` component is actually [utlizing](https://github.com/ampproject/amp-wp/pull/6320/files#diff-a47b24c70cb0eda5614a90acdb3a84ce27f313507d2a03d76ade03e5a6f2e626R24) the `wp.compose.useCopyToClipboard` hook.

I'm not entirely sure why Gutenberg is phasing out the `ClipboardButton` component. I suppose they want to have more freedom in how the button looks and behaves depending on the location in the UI.